test/sqllogictest/timestamp.slt

```diff
@@ -13,9 +13,6 @@ ALTER SYSTEM SET enable_binary_date_bin = true
 ----
 COMPLETE 0
 
-# Tests adapted from Postgresql
-# https://github.com/postgres/postgres/blob/master/src/test/regress/sql/timestamp.sql
-
```

Comment:
I removed this comment. The current version of the file behind that link looks very different from what's in here, so mentioning it here is more confusing than helpful.